packages/plugins/rudder_plugin_android/android/src/main/java/com/rudderstack/sdk/flutter/RudderSdkFlutterPlugin.java

```diff
@@ -229,6 +230,12 @@ public RudderConfig getRudderConfig(Map<String, Object> configMap) {
         .withTrackLifecycleEvents((Boolean) configMap.get("trackLifecycleEvents"))
         .withRecordScreenViews((Boolean) configMap.get("recordScreenViews"))
         .withControlPlaneUrl((String) configMap.get("controlPlaneUrl"));
+    String dataResidencyServer = (String) configMap.get("dataResidency");
+    if(dataResidencyServer.equals("US")){
```

Comment:
I would say if it is UE ten set to EU else US. US is our default one

Reply:
Sure, will update it